scripts/random.coffee

```diff
@@ -43,41 +31,19 @@ module.exports = (robot) ->
     if user == donny
       msg.send msg.random walter_quotes
 
-  robot.hear /pimmel/i, (msg) ->
-    msg.reply "Höhöhö, du hast Pimmel gesagt."
-
-  robot.hear /anyway/i, (msg) ->
-    msg.reply "How's your sex life?"
-
-  robot.hear /jehova/i, (msg) ->
-    msg.send "http://i.imgur.com/01PMBGj.gif"
-
-  robot.hear /frau/i, (msg) ->
-    msg.send ":lucas: Eh? :point_up:"
-
-  robot.hear /muss man wissen|fefe|sascha lobo|axel stoll/i, (msg) ->
-    msg.send "http://i.imgur.com/FmEyA8t.png"
-
-  robot.hear /schuh/i, (msg) ->
-    msg.send ":shoe::shoe:? Liegen wahrscheinlich noch bei Ben daheim."
-
-  robot.hear /you're tearing me apart|the room|tommy wiseau/i, (msg) ->
-    msg.send "http://i.giphy.com/pTrgmCL2Iabg4.gif"
-
-  robot.hear /citrix|35000|35.000|35k/, (msg) ->
-    msg.send ":moneybag::moneybag::moneybag:"
-
   robot.hear /matthias ist (.*)/, (msg) ->
     adj = msg.match[1].toLowerCase()
     if adj.indexOf("die tür") == -1 and adj.indexOf("die tuer") == -1
       msg.reply "Deine Mudda ist #{adj}!"
 
-  robot.hear /this is madness/i, (msg) ->
-    msg.send "Madness you say? THIS. IS. PATRI... MATTHIAS!"
+  robot.hear /bash me/i, (msg) ->
+    mgs.reply "http://bash.fsrleaks.de/?#{randomRange(1, 632)}"
```

Comment:
Of course there is ... see, thats what happens if I try to write code with words shorter than my autocomplete trigger ...